operate/client/e2e-playwright/test-fixtures.ts

```diff
@@ -68,6 +68,13 @@ const test = base.extend<
         return;
       }
 
+      // when using a baseURL containing domain + basePath, playwright expects a slash at the end of the baseURL when navigating and validating Urls
+      // https://playwright.dev/docs/api/class-browser#browser-new-context-option-base-url
+      // removing the slash here, as the used goto(url) below does not work with double slashes
+      if (baseURL && baseURL.endsWith('/')) {
+        baseURL = baseURL.slice(0, -1);
+      }
+
       // Important: make sure we authenticate in a clean environment by unsetting storage state.
       const page = await browser.newPage({storageState: undefined});
       await page.goto(`${baseURL}/login`);
```

Comment:
Hi @pedesen 
`new URL('/login', baseURL).toString()` uses absolute path would return `http://localhost:8080/login` instead of `http://localhost:8080/operate/login`
I can use absolute path `new URL('/operate/login', baseURL).toString()` but that would make the other CIs failing

Reply:
For transparency: We discussed to leave it as it is, because it was not possible to use `new URL` without breaking other things.